.travis.yml

```diff
@@ -1,16 +1,23 @@
+os: linux
 dist: bionic
 language: python
-matrix:
+jobs:
   include:
     - name: “Python 2.7 on xenial”
       python: "2.7"
       dist: xenial
     - name: “Python 2.7 on bionic”
       python: "2.7"
+    - name: “Python 3.8 make lint only”
+      python: "3.8"
+      install:
+        - pip install flake8
+        - make lint
+      script: true
```

Comment:
@cdrini I really don't like these `script: true` tests. Can't this be moved into `allowed failures` for clearer intent?

Reply:
script: true is a noop that overrides the script: tag below so that this run does not perform make i18n ; make test.

Reply:
Shouldn't the override be `script: make lint` so the result reflects the purpose of the test? I thought the `true` here is just the linux command to run for the job, not any specific Travis keyword?

Reply:
As far as I can tell, `install` is traditionally used for installing dependencies; `script` is used for building and testing. I guess linting makes sense to be in `script`? But that would make it inconsistent with the job below this. There, `lint` is run in the `install` step. My guess would be this was done on purpose to make it fail faster? Installing requirements/etc takes a while. But not sure.

Reply:
Correct.  I will make a new PR to move the lint command.